cabinet_wrapper.py

```diff
@@ -70,8 +70,18 @@ def _load_credentials(self, vault_name=None, account_id=None):
             print('Account not specified')
             exit()
 
-        self.password = getpass()
-        return self.open_vault(self.account_id, self.password, self.vault_name)
+        is_open = False
+        for i in range(3):
+            try:
+                self.password = getpass()
+                is_open = self.open_vault(self.account_id, self.password,
+                                          self.vault_name)
+                break
+            except Exception as e:
+                print("There was an error opening the vault, wrong password? "
```

Comment:
I tested it and looks good, but it has a tiny problem as it can be displayed in the following example:
```
$ python cli.py get 6-key
Password: 
There was an error opening the vault, wrong password? Try again or Ctrl+C to quit.
Password: 
There was an error opening the vault, wrong password? Try again or Ctrl+C to quit.
Password: 
There was an error opening the vault, wrong password? Try again or Ctrl+C to quit.
```
The error message should be printed only twice, as the third time the user enters a wrong password, the system should say "There was an error opening the vault, wrong password?", without the label `Try again or Ctrl+C to quit`, because the program is already terminated.